packages/dd-trace/src/metrics.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
       })
 
     try {
-      nativeMetrics = require('node-gyp-build')(path.join(__dirname, '..', '..', '..'))
+      nativeMetrics = util.loadMetrics()
```

Comment:
We don't need this anymore

Reply:
Well, the abstraction you added handles about that webpack thing, so i figured we could leave it like this

Reply:
ah, ok, I fix bugs without knowing it ? :p